code/modules/mob/living/carbon/xenomorph/death.dm

```diff
@@ -111,6 +111,8 @@
 		remains.icon_state = "gibbed-a-corpse-runner"
 	else if(isxenolarva(src))
 		remains.icon_state = "larva_gib_corpse"
+	else if(isxenohunter(src))
```

Comment:
Same

Reply:
Just do redefinitions.
`/mob/living/carbon/Xenomorph/Hunter/gib()` and so on.

Reply:
This is done